src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/email/AppellantSubmitAppealOutOfTimePersonalisationEmail.java

```diff
@@ -49,7 +49,7 @@ public String getReferenceId(Long caseId) {
     public Map<String, String> getPersonalisation(AsylumCase asylumCase) {
         requireNonNull(asylumCase, "asylumCase must not be null");
 
-        String dueDate = systemDateProvider.dueDate(14);
+        String dueDate = systemDateProvider.dueDate(28);
```

Comment:
maybe, it would be good to keep magic number in `application.yaml`

Reply:
I second that